conda/models/package_info.py

```diff
@@ -16,6 +17,10 @@
 
 
 class PackageInfo(Entity):
-
-    file = ListField(string_types)
-    # TODO: finish this
+    files = ListField(string_types)
+    has_prefix_files = ListField(string_types)
+    no_link = ListField(string_types)
+    soft_links = ListField(string_types)
+    index_json_record = ComposableField(Record)
+    icondata = StringField()
+    noarch = MapField()
```

Comment:
I don't like this format.  Thinking back to the `.psv` file, there should be one object for each row.  Let's discuss tomorrow.